pkg/agent/openflow/network_policy.go

```diff
@@ -1757,6 +1759,7 @@ func newFeatureNetworkPolicy(
 		enableDenyTracking:       enableDenyTracking,
 		enableAntreaPolicy:       enableAntreaPolicy,
 		category:                 cookie.NetworkPolicy,
+		connectUplinkToBridge:    connectUplinkToBridge,
```

Comment:
Do not get why we need this parameter in featureNetworkPolicy. Could not we use the flag in openflow client?

Reply:
We need to do some change for `func (f *featureNetworkPolicy) conjunctionActionFlow()`. Thus we should pass this parameter to `featureNetworkPolicy`